packages/back-end/src/api/fact-tables/updateFactTable.ts

```diff
@@ -69,3 +71,7 @@ export const updateFactTable = createApiRequestHandler(
     };
   }
 );
+
+export function needsColumnRefresh(changes: UpdateFactTableProps): boolean {
```

Comment:
We should add a way for this column refresh button to force a column refresh: https://github.com/growthbook/growthbook/blob/main/packages/front-end/components/FactTables/ColumnList.tsx#L97